scripts/ci.sh

```diff
@@ -35,7 +35,7 @@ setup_protoc() {
 	sudo mv protoc3/bin/* /usr/local/bin/
 	sudo mv protoc3/include/* /usr/local/include/
 	rm -f PROTOC_ZIP
-	go get -u github.com/golang/protobuf/protoc-gen-go \
+	GO111MODULE=off go get -u github.com/golang/protobuf/protoc-gen-go \
```

Comment:
We officially switched to Go1.13 (we needed this because of new error package).

I do not think that we need this Go1.11 ENV any more.

Reply:
Without this package didn't install globally. To remove this, I think we need to install them with `go install` instead of `go get`.

Reply:
Which Go version do you have on your machine? Can you switch to Go 1.13 and try?

Reply:
Yes, I will check it and let you know.

Reply:
I think here we want to install some packages in `GOPATH`, so we need to disable modules and install them. Another way is to change the current directory and then execute `go get` so it will install into `GOPATH`.

Reply:
@1995parham I would prefer either adding a comment that explains that, or actually doing that - because it is not self-documenting.

Adding a comment should be enough for now.

Reply:
We have an internal code-style convention that comments start with capital letter. This is neatpicking, but please correct the comment.